styles/pages/_home.scss

```diff
@@ -11,28 +11,26 @@
     @include filter(blur(5px));
   }
 
-  #bgFeature {
-    font-family: $font-heading;
-    font-size:.75em;
-    font-weight: 700;
-    text-decoration: underline;
-    text-transform: uppercase;
-    color: #ffffff;
-
-  }
-
-  #bgFeatureLocation {
-    font-family: $font-body;
-    font-size:.75em;
-    color: #ffffff;
-  }
-
   .bgFeatureBox {
-    float: right;
+    @include span (12);
     text-align: right;
-    clear: both;
+    padding-top: $topnav-logo-overhang / 2;
```

Comment:
I suppose the overhang will be leaving on scroll, so not sure how this will be affected.  I suppose a straight pixel amount makes more sense.  fxt.